src/app/products/share-products/view-share-product/view-share-product.component.html

```diff
@@ -3,7 +3,7 @@
     <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
     Edit
   </button>
-  <button mat-raised-button color="primary">
+  <button mat-raised-button color="primary" [routerLink]="['create']">
```

Comment:
```suggestion
  <button mat-raised-button color="primary>
```
Will merge this after 804, this is not the right place for the button.

Reply:
ok